File: src/API/CombatAnalysis.CommunicationAPI/Models/Post/PostModel.cs
namespace CombatAnalysis.CommunicationAPI.Models.Post;

public class PostModel
{
    public int Id { get; set; }

    public string Owner { get; set; }

    public string Content { get; set; }

    public int PostType { get; set; }

    public string Tags { get; set; }

    public DateTimeOffset When { get; set; }

    public int LikeCount { get; set; }

    public int DislikeCount { get; set; }

    public int CommentCount { get; set; }

    public string CustomerId { get; set; }
}

File: src/Web/CombatAnalysis.EnhancedWebApp/combatanalysis.enhancedwebapp.client/src/features/feed/types/PostModel.ts
export type PostModel = {
    id: number;
    content: string;
    tags: string;
    createdAt: Date;
    publicType: number;
    appUserId: string;
    likeCount: number;
    dislikeCount: number;
    commentCount: number;
    reaction: number;
}